.github/workflows/test.yaml

```diff
@@ -37,23 +37,32 @@ jobs:
             app: demo
             env: default
             path: ./samples/demo/app.bicep
-            args: --application demo
+            deployArgs: --application demo -p image=sampleregistry:5000/samples/demo
+            exposeArgs: --application demo
             uiTestFile: tests/demo/demo.app.spec.ts
             port: 3000
             container: demo
             enableDapr: false
+            images: samples/demo
+            directories: samples/demo/
           - name: dapr
             runOnPullRequest: true
             app: dapr
             env: default
             path: ./samples/dapr/dapr.bicep
+            deployArgs: -p frontendImage=sampleregistry:5000/samples/dapr-frontend -p backendImage=sampleregistry:5000/samples/dapr-backend
             enableDapr: true
+            images: samples/dapr-frontend,samples/dapr-backend
+            directories: samples/dapr/ui/,samples/dapr/nodeapp/
```

Comment:
I don't really like this hack but it seemed like the best way to pass multiple values. Arrays weren't working